requirements.txt

```diff
@@ -1,23 +1,46 @@
 aniso8601==9.0.1
```

Comment:
This comment goes wider than this PR but id like to suggest we use a requirement.lock for pip freeze requirements (a snapshot of the app and all the dependency versions) and keep requirements.txt to top level dependencies (we can use [pigar](https://pypi.org/project/pigar/) or [pipreqs](https://pypi.org/project/pipreqs/) for this).

Reply:
@gidsg this maybe related to your comment

Reply:
@Adam-W1 agreed, I believe we are already using `pip-compile` on some repos which implements the lock approach, so I think we should roll this out to all of them. CC @SentientHousePlant (we should make a story to track this work too)

Reply:
Created https://digital.dclg.gov.uk/jira/browse/FS-1137 to do this